ci/kokoro/windows/build-cmake-dependencies.ps1

```diff
@@ -48,7 +48,7 @@ if ($args.count -ge 1) {
     $vcpkg_flags=("--triplet", "${env:VCPKG_TRIPLET}")
 }
 $vcpkg_dir = "cmake-out\${vcpkg_base}"
-$vcpkg_version = "2021.04.30"
+$vcpkg_version = "2021.05.12"
```

Comment:
Ugh, I did not notice the build failures, looks like you need a call to `Set-Location ${vcpkg_dir}` around line 134.

Reply:
After trying things like `Push-Location` (which I think behaves the same as `Set-Location`), I couldn't get it to work. So I disabled manifest mode here with `$Env:VCPKG_FEATURE_FLAGS = "-manifests"`

Reply:
SGTM.